workshop/exercise-6/README.md

```diff
@@ -219,17 +236,17 @@ error=0
 
 ### 6. Push collection images
 
-To push the collection images, there is a little editing of one of the image list files (ensuring the registry location is correct), and then there is a release script that will do the actual pushing to the image registry. The two commands below will achieve this:
+To push the collection images, there is a release script that will do the actual pushing to the image registry. The current scripts have a minor issue with private registry login and tagging so first log in to the private registry, and then run the release script. The two commands below will achieve this:
 
 ```bash
-sed -i".bak" -e "s/$IMAGE_REGISTRY\///" ci/build/image_list && rm -f ci/build/image_list.bak
-./ci/release.sh
+echo $IMAGE_REGISTRY_PASSWORD | docker login -u $IMAGE_REGISTRY_USERNAME --password-stdin $IMAGE_REGISTRY
```

Comment:
This initially looks clumsy...but I agree it is the right way to do it. We should change ex3 to use the same format. As an aside, one classic issue is that the password token can expire...and this will fail. The worst case can be that ./ci/release.sh/ fails with unauthenticated....but it isn't obvious that it is referencing the IMAGE_REGISTRY_PASSWORD. Maybe this new approach below will avoid that.

Reply:
Ha - it happened to me when doing my tests, and it was actually the session login so even refreshing IMAGE_REGISTRY_PASSWORD with $(oc whoami -t) didn't help. The telltale sign is an authentication error when `release.sh` tries to push up the docker images. We might want to look for places to generally add this in a later update after more testing.

I updated ex3 with the simpler:
`oc whoami -t | docker login -u $(oc whoami) --password-stdin $IMAGE_REGISTRY`